### PR TITLE
Added Graph Algo in c++

### PR DESCRIPTION
**Pull request by Sahastracookcoder1**
- Added algo of articulation point in c++.

### CODE REVIEW

**Review by chanmol1999 — CHANGES_REQUESTED** [review]
Move this inside algorithms folder

File: Graph Algorithm/Articulation_Points/C++/articulationpoints.cpp [hunk 1]
```diff
@@ -0,0 +1,69 @@
+//this code can be used to find Articulation points in a Graph
+#include <bits/stdc++.h>
+using namespace std;
+
+int timee;// a variable to keep track of the time at which a particular vertex is encounterd
```
Comment (Bhavnaharitsa) [inline Graph Algorithm/Articulation_Points/C++/articulationpoints.cpp:5]: spelling check

File: Graph Algorithm/Articulation_Points/C++/articulationpoints.cpp [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+//visited - bool array used in DFS to know whether the node is visited or not
+//AP - bool array to tell whether ith vertex is a AP or NOT
+
+int disc[100] , low[100] , parent[100];
+//disc - discovery time of a vertex
+//low - exit time of a source
+//parent - array to tell the parent of ith vertex
+
+//finding all APs
+void printAP2(int source){
```
Comment (Bhavnaharitsa) [inline Graph Algorithm/Articulation_Points/C++/articulationpoints.cpp:18]: the function name can be printAp2

File: Graph Algorithm/Articulation_Points/C++/articulationpoints.cpp [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+			low[source] = min(low[i] , low[source]);
+			
+			if(parent[source] == source){
+				if(child > 1)	AP[source] = true;
+			}
+			else{
+				if(low[i] >= disc[source])	AP[source] = true; // IMP
+			}
+		}
+		else if(i != parent[source])	low[source] = min(low[i] , low[source]);
```
Comment (Bhavnaharitsa) [inline Graph Algorithm/Articulation_Points/C++/articulationpoints.cpp:37]: avoid unwanted space like i !

File: Graph Algorithm/Articulation_Points/C++/articulationpoints.cpp [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+	}
+	for(int i = 0 ; i < n ; i++){
+		if(AP[i])	cout << i << " ";
+	}
+	cout << endl;
+}
+
+int main(){
+	//n - no. of vertices , e - no. of edges
+	int n , e , u , v;
```
Comment (Bhavnaharitsa) [inline Graph Algorithm/Articulation_Points/C++/articulationpoints.cpp:60]: avoid single letter variables

**Comment by chanmol1999** [comment]
No Response going to close

**Pull request closed without merging**